src/js/osweb/classes/var_store.js

```diff
@@ -56,39 +59,38 @@ export default class VarStore {
    * @return {Boolean} - True if the variable is part of the store.
    */
   has (variable) {
-    // Check if the variable (property) is part of the class.
-    return this.hasOwnProperty(variable)
+    return this.inspect().contains(variable)

```

Comment:
`this.inspect()` returns an array, right? Is `contains()` a valid function nowadays? Shouldn't this be `Array.includes()`?